apps/projects/app/components/Panel/NewProject/RepoSelector.js

```diff
@@ -22,20 +22,26 @@ class Repo extends React.Component {
     repoSelected: -1,
   }
 
+  filterAlreadyAdded = repos => {
+    if (this.props.reposAlreadyAdded === undefined) return repos
+    const reposAlreadyAddedIds = this.props.reposAlreadyAdded.map(repo => repo.data._repo)
```

Comment:
If we just need `repo.data._repo` fields for that, maybe we should just pass an array of this field as props instead of the whole repos data.
This way we should just need to use `Array.filter()` function to filter out repos with the same id as `reposAlreadyAddedIds`

Reply:
Done.